### DISCUSSION TITLE
Same Columns, Multiple Dataframes

### DISCUSSION THREAD

**Discussion by wenger9** [sinaptik-ai/pandas-ai#442] [General]
Hi!

Let's say we have two dataframes, df1 and df2.

I have been wondering what happens when a query is posed that requires a reference to a column that exists in two separate dataframes?

For example, suppose df1 contains the following columns: ['name', 'job', 'sales'] while df2 contains ['actor', 'genre', 'sales']. df1 refers to a dataset of salepeople and their yearly sales and df2 refers to a dataset of actors and their yearly movie sales The query could be, "Who had the highest sales?". How does pandasai make the distinction?

Is there a way to ensure that pandasai will always (or almost always) be able to generate code that will execute against the right column references from the query?

Apologies if this is a poor example. Please let me know if additional examples would help!

**Comment by gventuri** [comment]
Hi @wenger9, very good question.
I would recommend being more specific on the prompt in such case. However, with version 1 we'll let you pass a custom name and custom instructions for each table, providing the LLM with a better context!

**Reply by wenger9** [reply]
That sounds great @gventuri! I will come back with a few more specific examples.

In the meantime, am I able to try the v1 pre-release?
